addons/web/static/src/core/scroller_service.js

```diff
@@ -35,12 +35,10 @@ export const scrollerService = {
                         ev.preventDefault(); // single hash in href are just a way to activate A-tags node
                         return;
                     }
-                    let matchingEl = null;
-                    try {
-                        matchingEl = document.querySelector(`.o_content #${href.value.substr(1)}`);
-                    } catch {
-                        // Invalid selector: not an anchor anyway
-                    }
+                    const selector = href.value.substr(1);
+                    const content = document.querySelector(".o_content");
+                    const el = selector.match(/\s/g) ? null : document.getElementById(selector);
```

Comment:
what if two elements with same id (i know its bad) are there, the first outside content, the other in?

Reply:
For now, the selector is ".o_content #selector". So only the element with id IN is selected.

Reply:
yes so here you change that: nothing will be selected